packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -900,6 +923,33 @@ void main() {
     });
   });
 
+  group('VideoPlayerOptions', () {
+    test('setMixWithOthers', () async {
+      final VideoPlayerController controller = VideoPlayerController.file(
+          File(''),
+          videoPlayerOptions: VideoPlayerOptions(mixWithOthers: true));
+      await controller.initialize();
+      expect(controller.videoPlayerOptions!.mixWithOthers, true);
+    });
+
+    for (final bool allowBackgroundPlayback in <bool>[true, false]) {
```

Comment:
nit: instead of looping and asserting 2 different things, this should be 2 different independent tests (factor out shared code if needed).  It's much easier to grok tests that are asserting absolutes instead of having the extra layer of indirection.

Reply:
Yeah, fair point.  Done.